cfgov/jinja2/v1/_includes/molecules/hero.html

```diff
@@ -12,12 +12,12 @@
 
    value:                  Object defined from a StreamField block.
 
-   value.heading:          A string containing title of module.
+   value.heading:          A string containing the title of the module.
 
-   value.body:             A string containing the content to display below
+   value.sub_heading:      A string containing the content to display below
                            the heading (typically a paragraph).
 
-   value.image:            An image object containing the image to be
+   value.large_image:      An image object containing the image to be
                            placed adjacent to the text.
 
    value.small_image:      An image object containing the alternate image
```

Comment:
Will these variables be alphabetized anywhere? If so we may want to make the naming general > specific and have `image_large` and `image_small` so they end up together.

Reply:
The variable order here matches the order in the Wagtail admin (in the screenshot above). I don't see why they'd ever be alphabetized.